### DISCUSSION TITLE
【BuildQuest】Raise your hand to join the hackathon team!

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#42] [Announcements]
Time: `04th March 2022 - SUN, MAR 13`

Attention: `need to stake 0.003ETH `

Links:

> https://hack.ethglobal.com/buildquest/home
>
> https://ethglobal.notion.site/Event-Rules-Code-of-Conduct-ebaa296461c44d1698d6a48e33665dbb

**Comment by fewwwww** [comment]
🙋

**Comment by Dream4ever** [comment]
🙋‍♂️

**Comment by leeduckgo** [comment]
GoodGood

**Comment by 0xKT** [comment]
🙋‍♂️
